src/action.c

```diff
@@ -321,6 +321,10 @@ action_arg_from_xml_node(struct action *action, const char *nodename, const char
 			action_arg_add_str(action, argument, content);
 			goto cleanup;
 		}
+		if (!strcmp(argument, "at_cursor")) {
```

Comment:
I think we usually use `CamelCase` for config args so `atCursor` might be more consistent.